File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/AppRoleModel.ts
import { AppRoleFeatureModel } from "./appRoleFeatureModel";
import { AppRoleUserModel } from "./appRoleUserModel";

export interface AppRoleModel {
    appRoleId: number;
    appRoleCode: string;
    appRoleNameEn: string;
    appRoleNameAr: string;
    icon: string;
    color: string;
    displayOrder: number | null;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}

File: SofiForce.Models/Models/EntityModels/AppRoleModel.cs
using System;

namespace Models
{
    public class AppRoleModel
    {
        public int AppRoleId { get; set; }
        public string AppRoleCode { get; set; }
        public string AppRoleNameEn { get; set; }
        public string AppRoleNameAr { get; set; }
        public string icon { get; set; }
        public string color { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public bool? FullAccess { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
